_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1210 @@
+* June
+  * [Calculating the selfish mining danger threshold](#selfishmining)
+  * [Fingerprinting nodes using addr messages](#fingerprinting)
+  * [Garbled locks](#garbledlocks)
+* July
+  * [Chain code delegation](#ccdelegation)
+* August
+  * [Utreexo draft BIPs](#utreexo)
+  * [Lowering the minimum relay feerate](#minfeerate)
+  * [Peer block template sharing](#templatesharing)
+  * [Differential fuzzing of Bitcoin and LN implementations](#fuzzing)
+* September
+  * [Details about the design of Simplicity](#simplicity)
+  * [Partitioning and eclipse attacks using BGP interception](#eclipseattacks)
+* October
+  * [Theoretical limitations on embedding data in the UTXO set](#arbdata)
+  * [Channel jamming mitigation simulation results and updates](#channeljamming)
+* November
+  * [Comparing performance of ECDSA signature validation in OpenSSL vs. libsecp256k1](#secpperformance)
+  * [Multiple discussions about restricting data](#restrictingdata)
+  * [Modeling stale rates by propagation delay and mining centralization](#stalerates)
+  * [BIP3 and the BIP process](#bip3)
+  * [Bitcoin Kernel C API introduced](#kernelapi)
+* December
+* Featured summaries
+  * [Vulnerabilities](#vulns)
+  * [Quantum](#quantum)
+  * [Soft fork proposals](#softforks)
+  * [Stratum V2](#stratumv2)
+  * [Major releases of popular infrastructure projects](#releases)
+  * [Bitcoin Optech](#optech)
+
+---
+
+## January
+
+{:#chilldkg}
+- **Updated ChillDKG draft:** Tim Ruffing and Jonas Nick updated their work on
+  a distributed key generation protocol (DKG) for use with the [FROST][news
+  frost bip] [threshold signature][topic threshold signature] scheme. ChillDKG
+  aims to provide similar recoverability features to existing descriptor
+  wallets.
+
+{:#offchaindlcs}
+- **Offchain DLCs:** Developer Conduition [posted about][news offchain dlc] a
+  new offchain DLC ([discreet log contract][topic dlc]) mechanism that enables
+  participants to collaborate on the creation and extension of a DLC factory
+  which allows iterative DLCs that roll along until one party chooses to
+  resolve on chain. This contrasts with [prior work][news dlc channels] on
+  offchain DLCs which required interaction at each roll of the contract.
+
+{:#compactblockstats}
+- **Compact block reconstructions:** January also saw the first of several items
+  in 2025 that revisited [previous research][news315 compact blocks] into how
+  effectively Bitcoin nodes reconstruct blocks using [compact block relay][topic
+  compact block relay] (BIP152), updating previous measurements and exploring
+  potential refinements. Updated statistics [published in January][news339
+  compact blocks] showed that compact blocks continued to reconstruct
+  successfully at high rates, while also identifying that during full-mempool
+  scenarios, nodes more frequently needed to request missing transactions, in
+  this case, orphan transactions.
```

Comment:
@reardencode: 

> showed that compact blocks continued to reconstruct successfully at high rates

That's not really a takeaway I have from the write-up nor the Delving posts, with e.g. the Delving chart showing pretty terrible reconstruction in 2024-12. I'd suggest to just leave that out?

Commit to cherry-pick: https://github.com/stickies-v/bitcoinops.github.io/commit/cd542b6d64339f5edc6e5eb91a871f5049edba1f